coralproject/talk
# [CORL-282] Handle server errors in client

## What does this PR do?
- Added Talk Errors `COMMENT_BODY_TOO_SHORT` and 
`COMMENT_BODY_EXCEEDS_MAX_LENGTH`
- Added enum `ERROR_TYPES`
- Handle server errors and proof it with tests 😃 (in Admin Configuration, Post / Edit / Reply Comment, and report a comment) 

## How to test the PR?
- E.g. to test Post / Edit / Reply a comment, simply submit a body length that does not adhere to min and max. As client side validations for the comment length has not yet been implemented this should trigger a server error.
